docs/configuration-setup.rst

```diff
@@ -26,7 +26,7 @@ Or as part of a configuration file:
    cluster = PBSCluster()
 
 For more information on handling configuration files see `Dask configuration
-documentation <https://dask.org/en/latest/configuration.html>`_.
+documentation <https://dask.org/en/latest/configurations.html>`_.
```

Comment:
Thanks for your contribution. I think this should actually be: `http://docs.dask.org/en/latest/configuration.html`

Reply:
Ah ok, I just saw the dead link and didn't look too deeply...fixing now